### PR TITLE
IN - Mostrar fecha y hora del mapa de cama

### PR DESCRIPTION
**Pull request by MarianoCampetella**
### Requerimiento
https://proyectos.andes.gob.ar/browse/IN-431

### Funcionalidad desarrollada 

1. Se agrego la fecha junto con la hora dentro del plex-title

### UserStory llegó a completarse

- [X] Si
- [ ] No
- [ ] No corresponde

### Requiere actualizaciones en la base de datos

- [ ] Si
- [X] No

### Requiere actualizaciones en la API

- [ ] Si
- [X] No

### Requiere actualizaciones en andes-test-integracion

- [X] Si   https://github.com/andes/andes-test-integracion/pull/429
- [ ] No

### CODE REVIEW

**Review by palita1991 — APPROVED** [review]

**Comment by martinebucarey** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [6362](http://condorpiedra.andes.gob.ar:85/6362/mochawesome.html)
CYPRESS RUN: [5247](https://dashboard.cypress.io/projects/xr7gft/runs/5247/specs)
TEST START: 2022-05-04T18:18:07.779Z
TOTAL: 337
SUCCESS: 322
FAIL: 14
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [6369](http://condorpiedra.andes.gob.ar:85/6369/mochawesome.html)
CYPRESS RUN: [5254](https://dashboard.cypress.io/projects/xr7gft/runs/5254/specs)
TEST START: 2022-05-05T11:48:59.428Z
TOTAL: 406
SUCCESS: 403
FAIL: 2
SKIPPED: 1

File: src/app/apps/rup/mapa-camas/views/mapa-camas-capa/mapa-camas-capa.component.html [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-                <plex-dropdown *ngIf="mapaCamasService.capa === 'estadistica' && permisosMapaCamasService.censo"
-                               size="sm" label="VER CENSO" type="primary" class="d-inline-block mr-1" [right]="true"
-                               [items]="itemsCensoDropdown">
-                </plex-dropdown>
-                <plex-button type="info" size="sm" (click)="onVisualizar()" tooltip="Mapa Visual de Camas">
-                    <plex-icon name="atom"></plex-icon>
-                </plex-button>
-            </plex-title>
+            <ng-container *ngIf="fecha$ | async as fecha">
+                <plex-title titulo="Mapa de camas - {{fecha | date: 'dd/MM/yyyy'}}   {{fecha | date: 'HH:mm'}}">
```
Comment (martinebucarey) [inline src/app/apps/rup/mapa-camas/views/mapa-camas-capa/mapa-camas-capa.component.html:5]:
```suggestion
                <plex-title titulo="Mapa de camas - {{fecha | date: 'dd/MM/yyyy HH:mm'}}">
```

**Review by martinebucarey — CHANGES_REQUESTED** [review]

**Review by AgosLizzi — APPROVED** [review]

**Comment by martinebucarey** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [6372](http://condorpiedra.andes.gob.ar:85/6372/mochawesome.html)
CYPRESS RUN: [5257](https://dashboard.cypress.io/projects/xr7gft/runs/5257/specs)
TEST START: 2022-05-05T12:28:42.370Z
TOTAL: 406
SUCCESS: 403
FAIL: 2
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [6375](http://condorpiedra.andes.gob.ar:85/6375/mochawesome.html)
CYPRESS RUN: [5260](https://dashboard.cypress.io/projects/xr7gft/runs/5260/specs)
TEST START: 2022-05-05T14:34:02.990Z
TOTAL: 332
SUCCESS: 306
FAIL: 25
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [6376](http://condorpiedra.andes.gob.ar:85/6376/mochawesome.html)
CYPRESS RUN: [5261](https://dashboard.cypress.io/projects/xr7gft/runs/5261/specs)
TEST START: 2022-05-05T14:53:21.234Z
TOTAL: 402
SUCCESS: 400
FAIL: 1
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [6377](http://condorpiedra.andes.gob.ar:85/6377/mochawesome.html)
CYPRESS RUN: [5262](https://dashboard.cypress.io/projects/xr7gft/runs/5262/specs)
TEST START: 2022-05-05T15:25:02.203Z
TOTAL: 406
SUCCESS: 391
FAIL: 14
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [6378](http://condorpiedra.andes.gob.ar:85/6378/mochawesome.html)
CYPRESS RUN: [5263](https://dashboard.cypress.io/projects/xr7gft/runs/5263/specs)
TEST START: 2022-05-05T15:38:29.548Z
TOTAL: 365
SUCCESS: 352
FAIL: 12
SKIPPED: 1

**Pull request merged**